ledger/sandbox/BUILD.bazel

```diff
@@ -225,6 +225,7 @@ da_scala_library(
         "//daml-lf/archive:daml_lf_archive_reader",
         "//daml-lf/archive:daml_lf_dev_archive_java_proto",
         "//daml-lf/data",
+        "//daml-lf/interpreter",
```

Comment:
`//daml-lf/interpreter` is where Speedy lives, right? Does it make sense for it to be a direct dependency of the sandbox? Any chance we can keep those separated?

Reply:
Moving into the sandbox in the current state, it is clearly not an option. 
However, If you are keen on dropping this dependency, I could move the type definition in "//daml-lf/transaction" (which is a fair dependency) in an upcomming PR.

Reply:
Makes sense thanks, I created https://github.com/digital-asset/daml/issues/6260 so that you can use it to track it in the the upcoming PR you mentioned.